tests/bookApiMock.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test } from "../src/fixtures/baseFixtures";
+import { expect } from "@playwright/test";
+
+test.beforeAll('Fetch Books Api, response must be ok', async ({ userApiRequest }) => {
+    const response = await userApiRequest.fetch('https://demoqa.com/BookStore/v1/Books');
+    expect.soft(response.ok()).toBe(true);
+    expect.soft(response.status()).toBe(200);
+});
+
+test('Mock response body, validate visible via FE', async ({ page }) => {
+
+    await page.route('https://demoqa.com/BookStore/v1/Books', async route => {

```

Comment:
you can use smth like that, in this case this URL will be suitable for diff env
```suggestion
    await page.route('**/*/v1/Books', async route => {

```